db/seeds.rb

```diff
@@ -6,8 +6,44 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 Compliment.find_or_create_by(message: "You rock")
-Organization.find_or_create_by(
+
+organizations = [{
   name: 'RailsBridge',
+  description: 'RailsBridge workshops are a free and fun way to get started or level up with web technologies.',
```

Comment:
Line is too long. [112/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.